lasio/reader.py

```diff
@@ -751,6 +751,47 @@ def read_header_line(line, pattern=None, section_name=None):
     """
```

Comment:
It would help to have a mention of the `pattern` and `section_name` keyword arguments by referring the reader to the `configure_and_match_pattern()` docstring instead.

Reply:
I'm adding section_name and pattern argument information to read_header_line() docstring so there is no-jumping around looking for the information.  Also the Return value description on configure_and_match_pattern() will be updated to state that it is an array of regular expression strings (patterns).